src/product/ProductEdit.tsx

```diff
@@ -106,6 +123,54 @@ const ProductEditForm = () => {
           />
         </Box>
       </FormTab>
+
+      <FormTab label="reviews" path="reviews">
+        <ReferenceManyField
+          perPage={10}
+          reference="reviews"
+          target="product_id"
+          pagination={
+            <Pagination sx={{ alignSelf: "end", color: "text.secondary" }} />
+          }
+        >
+          <Datagrid
+            empty={
+              <Grid container justifyContent="center">
+                <Grid item>
+                  <Typography>No review yet on this poster</Typography>
+                </Grid>
+              </Grid>
+            }
+            bulkActionButtons={false}
+            sx={{
+              "& .RaDatagrid-headerCell": { fontWeight: "bold" },
+              width: "100%",
+            }}
+          >
+            <DateField source="date" />
+            <ReferenceField source="customer_id" reference="customers">
+              <CustomerField />
+            </ReferenceField>
+            <WrapperField label="Rating" sortBy="rating">
+              <Rating />
+            </WrapperField>
```

Comment:
I didn't even know about this `WrapperField`. I think you don't need it though. Just accept both props on your component event though you don't use them internally